meta-facebook/yv35-hd/src/ipmi/plat_ipmi.c

```diff
@@ -386,3 +387,27 @@ void OEM_1S_GET_HSC_STATUS(ipmi_msg *msg)
 	msg->completion_code = CC_SUCCESS;
 	return;
 }
+
+void OEM_1S_GET_4BYTE_POST_CODE(ipmi_msg *msg)
```

Comment:
This would mean that this code would be duplicated for any AMD based platform.
Can you provide more justification of moving it out of common?